packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -791,8 +791,8 @@ contract Deploy is Deployer {
                 vm: IBigStepper(artifacts.mustGetAddress("MipsSingleton")),
                 weth: weth,
                 anchorStateRegistry: IAnchorStateRegistry(artifacts.mustGetAddress("AnchorStateRegistryProxy")),
-                l2ChainId: cfg.l2ChainID()
-            })
+                l2ChainId: 0 // Unused Param on SuperDisputeGame
```

Comment:
Doesn't this result in a revert for `NoChainIdNeeded` when trying to create a Super DG?

Reply:
Should've been ` != 0` for chainId checks, my bad